### PR TITLE
Add web3 + ethers.js in standard detection guides

### PR DESCRIPTION
**Pull request by rowkee**
Screenshot 2024-07-08 at 11 49 15

Making the following changes:
- adding ether and web3 code snippets for `Detect Universal Profile data keys`, `Example - Detect Digital Asset data keys ` and `Interface Detection `
- created new class for code snippet tabs for erc725.js

### CODE REVIEW

File: docs/learn/standard-detection.md [hunk 1]
```diff
@@ -21,15 +24,15 @@ You can also use the [🔎 ERC725 Inspect Tool](https://erc725-inspect.lukso.tec
...
+The following code snippets require the following libraries to be installed:
 
 ```shell
 npm install web3 @erc725/erc725.js @lukso/lsp-smart-contracts
 ```
 
 ## Metadata Detection
 
-You can verify if a contract contains a specific set of ERC725Y keys (= **metadata**) by checking the value stored under the ERC725Y storage key `SupportedStandards:{StandardName}` using the [erc725.js](../tools/erc725js/getting-started.md) library.
+You can verify if a contract contains a specific set of [ERC725Y](../standards/generic-standards/lsp2-json-schema) keys by checking the value stored under the ERC725Y storage key `SupportedStandards:{StandardName}` using the [erc725.js](../tools/erc725js/getting-started.md) library.
```
[the remark this answers is not in the document]
Comment (CJ42) [reply docs/learn/standard-detection.md:35]: This should be linked to the ERC725Y page, not LSP2

File: docs/learn/standard-detection.md [hunk 2]
```diff
@@ -39,50 +42,176 @@ You can verify if a contract contains a specific set of ERC725Y keys (= **metada
 
 Similar to the [Read Profile Data Guide](./universal-profile/metadata/read-profile-data.md), you can use the [`getData()`](../tools/erc725js/methods.md#getdata) function to check if the contract has a specific metadata standard like [LSP3 Profile](../standards/universal-profile/lsp3-profile-metadata), [LSP4 Digital Asset](../standards/tokens/LSP4-Digital-Asset-Metadata) or a [LSP9 Vault](../standards/universal-profile/lsp9-vault).
 
-### Example - Detect Universal Profile data keys
+### Example 1 - Detect LSP3 Profile data keys
 
-```js
-import { ERC725 } from '@erc725/erc725.js';
-import lsp3ProfileSchema from '@erc725/erc725.js/schemas/LSP3ProfileMetadata.json';
+<!-- For web3.js + ethers.js, we do not need to import the schema from erc725.js -->
```
Comment (CJ42) [inline docs/learn/standard-detection.md:47]: need to remove this comment

File: docs/learn/standard-detection.md [hunk 3]
```diff
@@ -39,50 +42,176 @@ You can verify if a contract contains a specific set of ERC725Y keys (= **metada
...
+  
+  <TabItem value="erc725.js" label="erc725.js"  attributes={{className: "tab_erc725"}}>
+
+    ```javascript
+    import lsp3ProfileSchema from '@erc725/erc725.js/schemas/LSP3ProfileMetadata.json';
+    import { SupportedStandards } from '@lukso/lsp-smart-contracts';
+    import { ERC725 } from '@erc725/erc725.js';
+
+    const erc725js = new ERC725(
+    lsp3ProfileSchema,
```
Comment (CJ42) [inline docs/learn/standard-detection.md:115]: there should be an indent here.

**Review by CJ42 — CHANGES_REQUESTED** [review]

File: docs/learn/standard-detection.md (hunk 3 above)
Comment (rowkee) [reply docs/learn/standard-detection.md:115]: I think it's prettier changing it, and I can't stop it. I tried using  and

File: docs/learn/standard-detection.md [hunk 4]
```diff
@@ -39,50 +42,172 @@ You can verify if a contract contains a specific set of ERC725Y keys (= **metada
...
+  <TabItem value="ethers" label="ethers" attributes={{className: "tab_ethers"}}>
 
-const erc725js = new ERC725(
-  lsp3ProfileSchema,
-  '0x70DACA6E328dfBc8801b8efa4de9e579cC151b86', // Universal Profile contract address
-  'https://4201.rpc.thirdweb.com',
-  {
-    ipfsGateway: 'https://api.universalprofile.cloud/ipfs',
-  },
+    ```javascript
```
Comment (mustafademiray) [inline docs/learn/standard-detection.md:50]: could you try changing this to typescript for better highlighting?

**Review by CJ42 — APPROVED** [review]

**Pull request merged**